spec/index.html

```diff
@@ -288,7 +288,7 @@ <h3>Object Lists</h3>
   </section>
 
   <p>
-    There are three four of <em>RDF Term</em> defined in RDF Concepts:
+    There are four <a data-cite="RDF12-CONCEPTS#dfn-rdf-term">RDF terms</a> defined in RDF Concepts:
     <a data-cite="RDF12-CONCEPTS#dfn-iri">IRIs</a> (Internationalized Resource Identifiers),
     <a data-cite="RDF12-CONCEPTS#dfn-literal">literals</a>,
     <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank nodes</a>, and.
```

Comment:
Note: errant `.` at end of line 294, which should replace `,` and end of line 295. (Alternatively, `,` at end of line 295 could be changed to `;`)